utilityMethods.js

```diff
@@ -133,35 +133,42 @@ const spawnFish = map => {
   for (let row = 0; row < map.length; row++) {
     for (let col = 0; col < map[row].length; col++) {
       const x = map[row][col]
-      if (x < 47 && x >= 30) {
```

Comment:
yo, the ranges of x in spawnFish when populating the array of shallows / open ocean / deep refer to the heights in the map. The double for loop was just for populating each depth array. The frequency was originally determined by the % statements -- so if the modulus is bigger, there's a lower chance of a fish spawning. Adjusting the number of tiles in each array works, demonstrably, but it's mixing our metaphors.